### PR TITLE
 benchmark/radosbench: add "read_only", "read_time" and "write_time" options

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
being tested at http://pulpito.ceph.com/kchai-2019-07-02_07:44:20-rados-wip-kefu2-testing-2019-07-01-0133-distro-crimson-mira/

File: benchmark/radosbench.py [hunk 1]
```diff
@@ -60,54 +63,54 @@ def initialize(self):
...
 
     def run(self):
         super(Radosbench, self).run()
-        
+
+        do_prefill = self.prefill_time or self.prefill_objects
+        # sanity tests
+        if self.read_only and self.write_only:
+            logger.error('Both "read_only" and "write_only" are specified, '
+                         'but they are mutual exclusive.')
```
Comment (neha-ojha) [inline benchmark/radosbench.py]: nit: mutually

Comment (tchaikov) [reply benchmark/radosbench.py]: fixed and updated.

**Comment by tchaikov** [comment]
@neha-ojha i will try to document these options in a follow-up PR.

**Pull request merged**